PSKoans/Data/Advice/Fun/CatFunExample.Advice.json

```diff
@@ -0,0 +1,22 @@
+{
+    "Title": "Cat Fun Example",
```

Comment:
```suggestion
    "Title": "Fun with Cat Facts",
```

Reply:
Title will be changed on both in the next commit 🦄